fanuc_r2000ib_support/launch/robot_interface_downloading_r2000ib165f.launch

```diff
@@ -0,0 +1,19 @@
+<!--
+  Manipulator specific version of 'robot_interface_streaming.launch'.
+
+  Usage:
+    robot_interface_streaming_r2000ib165f.launch robot_ip:=<value>
+-->
+<launch>
+  <arg name="robot_ip" />
+  <arg name="enable_control_hand" default="true" />
+<!--  <arg name="use_bswap" default="true" />-->
```

Comment:
Could you uncomment this?

Reply:
Same for all other places where you've commented it.

Reply:
I copied and adapted your launchfiles from r2000ic165f now, sorry for the oversight